### ISSUE TITLE
Update the "How to run" section

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#6]
### Who

- [ ] Anonymous visitor
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [x] Administrator
- [x] Developer

### Needs what

Needs to know how to run the project.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #4

### Blockers

This issue is blocked by the following issue(s): #5
